modules/nanoflann/1.5.4/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/jlblancoc/nanoflann/archive/refs/tags/v1.5.4.tar.gz",
```

Comment:
Doesn't this need something like
```suggestion
    "url": "https://github.com/jlblancoc/nanoflann/archive/refs/tags/v1.5.4.tar.gz",
    "strip_prefix": "nanoflann-v1.5.4",
```

Reply:
You are absolutely right. This got lost.... Should be without the `v`, though. Will fix

Reply:
Fixed remaining issues. Thanks!